corehq/apps/hqwebapp/static/app_manager/less/case_tile_preview.less

```diff
@@ -13,6 +13,10 @@
         // TODO: to be truly adorable, add transitions on changes to grid style, this doesn't work
         transition: 2s;
 
+        &.odd-tab {
+            background-color: darken(@cc-bg, 3);
```

Comment:
Can we reuse the colors that Jonathan used for the groups?

Reply:
These ones? https://github.com/dimagi/commcare-hq/pull/33801

I like the thought of reusing colors, but those are so dark they seem a bit overwhelming:
<img width="1004" alt="Screen Shot 2024-01-18 at 2 10 59 PM" src="https://github.com/dimagi/commcare-hq/assets/1486591/20c2bd70-745b-48fa-b54f-68c5552d3050">

Reply:
Oh. Agreed.